client/src/plus/search-filter/index.tsx

```diff
@@ -107,7 +234,16 @@ export default function SearchFilter({
             />
           </DropdownMenu>
         </DropdownMenuWrapper>
-      ) : null}
+      ))}
+
+      <Search
+        isDisabled={isDisabled}
+        name="terms"
+        placeholder="Filter by keyword"
+        value={terms}
+        onBlurHandler={() => setSelectedTerms(camelUnwrap(terms))}
```

Comment:
Again perhaps so comment for why this is needed :) .

Reply:
Resolved via 6019f8f71379f9bfe1ff7d943600a9de667c9284.